build.gradle

```diff
@@ -695,7 +695,15 @@ if (HUDSON_JOB_NAME == "not_hudson") {
     relSuffix = "-internal"
     relOpt = "-${buildTimestamp}"
 } else {
-    relSuffix = IS_MILESTONE_FCS ? "" : jfxReleaseSuffix
+    if (IS_MILESTONE_FCS) {
+        if (jfxExperimentalReleaseSuffix != "") {
+            fail("An experimental release branch should not be used for FCS build")
+        }
+        relSuffix = ""
+    } else {
+        relSuffix = jfxExperimentalReleaseSuffix != "" ?
+                        jfxExperimentalReleaseSuffix : jfxReleaseSuffix;
```

Comment:
One thought I had: we might also want to set `relOpt = "-${buildTimestamp}"` here like we do for internal builds so we always see a time stamp for experimental builds. What do you think?

Reply:
I agree with that. It is often useful to have timestamp info in the builds during development.
I know I'm a broken record on this topic, but if possible, it would be great if (parts of) this logic can be moved into a separate file in order to make the build.gradle smaller and more readable. I'm not insisting on this for this PR, but I think it is worth thinking about this whenever changes to build.gradle are made.

Reply:
> I know I'm a broken record on this topic, but if possible, it would be great if (parts of) this logic can be moved into a separate file in order to make the build.gradle smaller and more readable.

Also at the risk of sounding like a broken record, https://github.com/openjdk/jfx/pull/1232 has been waiting to start this process for a while now and it's stuck waiting for explanations on how publishing works, something only Oracle and Gluon do. If someone can help Jendrik Johannes there then the ball will get rolling. I tried to split the big build file into separate files for each module, but it's entangled with the code in that PR. Completing that PR might also remove the need for the JavaFX plugin.

Reply:
> > I know I'm a broken record on this topic, but if possible, it would be great if (parts of) this logic can be moved into a separate file in order to make the build.gradle smaller and more readable.
> 
> Also at the risk of sounding like a broken record, #1232 has been waiting to start this process for a while now and it's stuck waiting for explanations on how publishing works, something only Oracle and Gluon do. If someone can help Jendrik Johannes there then the ball will get rolling. I tried to split the big build file into separate files for each module, but it's entangled with the code in that PR. Completing that PR might also remove the need for the JavaFX plugin.

Fair remark. I just commented on the PR.

Reply:
Added `relOpt = "-${buildTimestamp}" `